recipes/libobjcryst/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "libobjcryst" %}
+{% set version = "2017.1.1" %}
+{% set sha256 = "605710be928274a0a73efeda0bf76a8a32c8f5fcedce27b25fd96a6cc67fcf26" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/diffpy/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - scons 2.*
+    - boost 1.61.0
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libObjCryst${SHLIB_EXT}
+
+about:
+  home: https://github.com/diffpy/libobjcryst/
+  license: GPL2
+  license_family: GPL
+  license_file: LICENSE.txt
+  summary: 'ObjCryst++ packaged for installation as a shared library.'
+
+  description: |
+    ObjCryst++ is Object-Oriented Crystallographic Library for C++ developed
+    by Vincent Favre-Nicolin. libobjcryst is a mirror of the ObjCryst++
+    sources from https://sourceforge.net/projects/objcryst, expanded with
+    SCons build scripts to make it easier to install as a system shared
+    library. libobjcryst does not include GUI related files from ObjCryst++.
+    This distribution contains required sources from cctbx and newmat that are
+    used in ObjCryst++.
+  doc_url: https://github.com/diffpy/libobjcryst
+  dev_url: https://github.com/diffpy/libobjcryst
+
+extra:
+  recipe-maintainers:
+    - CJ-Wright
+    - pavoljuhas
```

Comment:
@pavoljuhas, are you ok with being added as a maintainer for this recipe

Reply:
@isuruf - Yes, that is OK with me.